.env.example

```diff
@@ -0,0 +1,9 @@
+# Environment variables
+# below for pizza advanced
+#WORKSPACE_ID= 52df88d6-a482-4b8e-80bd-baaf24de55c5
+# below for pizza simple
+#WORKSPACE_ID=22fe9213-6691-42d4-b234-71f4238f074b
+# below for conversation simple.
+WORKSPACE_ID= 33412b8a-c610-4eb2-bcd0-42ac716b3f78
+CONVERSATION_USERNAME=a4e698f7-24ad-4ce8-b104-4237592e43de
```

Comment:
I'd also like it if we were consistent with our original env.sample. I think we are starting to get a variety of example env name though